docs/choosing_clusters.md

```diff
@@ -54,10 +54,8 @@ In the Docker For Mac preferences, click
 [Kind](https://kind.sigs.k8s.io/) runs Kubernetes inside a Docker container.
 
 The Kubernetes team uses Kind to test Kubernetes itself. But its fast startup
-time also makes it a good solution for local dev. Follow these instructions for
-setting up Kind with Tilt:
-
-[Kind Setup Instructions](https://github.com/windmilleng/kind-local)
```

Comment:
ooh, can you leave this as-is? i think it's pretty important that there's a strong call to action